pkg/cvo/cvo_test.go

```diff
@@ -2671,6 +2673,9 @@ func TestOperator_availableUpdatesSync(t *testing.T) {
 				if optr.availableUpdates != nil && optr.availableUpdates.Upstream == "http://localhost:8080/graph" {
 					optr.availableUpdates.Upstream = s.URL
 				}
+				if tt.wantUpdates != nil && tt.wantUpdates.Condition.Status == configv1.ConditionTrue {
+					tt.wantUpdates.Condition.Message = fmt.Sprintf("Successfully retrieved updates from %s", s.URL)
+				}
```

Comment:
I am not sure how is this going help?

Reply:
The error message includes the upstream URI.  In these tests, we construct the slice of test-cases before we create the upstream service, so at this point, now that we know the upstream URI and can construct the expected message, I'm inserting that message in the test-case so that the later "does the actual message match the test-case's expected message?" logic can confirm that the upstream URI is included.